.github/workflows/deploy.yml

```diff
@@ -35,7 +35,9 @@ jobs:
       - name: Login to ibmcloud
         env:
           API_KEY: ${{ secrets.API_KEY }}
-        run: ibmcloud login -a "https://cloud.ibm.com" -u apikey -p "$API_KEY" -o "carbon-design-system" -s "production" -r "us-south"
+        run:
+          ibmcloud login -a "https://cloud.ibm.com" -u apikey -p "$API_KEY" -o
```

Comment:
If prettier is formatting this line this way, we may need to use the `|` character for multiline just to make sure the bash command works as expected 👀

Reply:
@joshblack should I just include that character at the end of the first line? I thought the multiline character was `\`? then again, bash is not my strong-suit so I may be making things up lol